recipes-bsp/sgx/sgx-sdk-cross_2.12.bb

```diff
@@ -25,8 +25,8 @@ EXTRA_OEMAKE += "OCAMLLIB='${STAGING_LIBDIR_NATIVE}/ocaml' -C sdk signtool edger
 ### install ###
 
 do_install () {
-    install -d ${D}${sgxdirprefix}${sgxrootdir}
-    cp -r ${RECIPE_SYSROOT}${sgxsdkpath} ${D}${sgxdirprefix}${sgxsdkpath}-cross
+    ${RECIPE_SYSROOT}${sgxrootdir}/package/sgx_linux_x64_sdk_2.12.100.3.bin --prefix="${D}${sgxdirprefix}${sgxrootdir}"
```

Comment:
Please mark this recipe as only compatible with x64 build host, or modify it to use qemu user mode emulation.

Reply:
done